src/platform/site-wide/announcements/containers/Announcement.jsx

```diff
@@ -66,6 +66,7 @@ export class Announcement extends Component {
         announcement={announcement}
         dismiss={dismiss}
         isLoggedIn={this.props.isLoggedIn}
+        useSSOe={this.props.useSSOe}
```

Comment:
You can pass additional props to your component via the config, `src/platform/site-wide/announcements/config/index.js`, and get access to the whole config object via `this.props.announcement.propName`

Reply:
this prop, `useSSOe` comes from mapStateToProps.  Is it possible to add a prop in the config if it needs to be computed with the state object?

Reply:
Ah you're right, sorry!